velox/exec/Exchange.cpp

```diff
@@ -156,6 +147,15 @@ class LocalExchangeSource : public ExchangeSource {
         });
   }
 
+  bool shouldRequestLocked() override {
```

Comment:
Any particular reason to move this method?

Reply:
The order is messed when I copy back the code. Let me change it